### PR TITLE
Create observability-platform-cost-estimate

### PR DESCRIPTION
**Pull request by Rotfuks**
Introduce new dashboard for estimating observability platform costs.

This PR

- adds/changes/removes etc
- screenshots before:
- screenshots after:

### Checklist

- [ ] Update changelog in CHANGELOG.md in an end-user friendly language.

### CODE REVIEW

**Comment by QuantumEnigmaa** [comment]
Could you show screenshots from your dashboard pls ? :pray:

**Comment by Rotfuks** [comment]
sure sure, here's the dashboard @QuantumEnigmaa

File: helm/dashboards/charts/private_dashboards_mz/dashboards/shared/private/observability-platform-cost-estimate [hunk 1]
```diff
@@ -0,0 +1,830 @@
...
+    ]
+  },
+  "time": {
+    "from": "now-7d",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "browser",
+  "title": "Observability Platform: Cost Estimate",
+  "uid": "fe1n1g0gwo8aob",
```
Comment (hervenicol) [inline helm/dashboards/charts/private_dashboards_mz/dashboards/shared/private/observability-platform-cost-estimate:827]: We advise for a "human readable UID" so creating links to dashboards is easier.
```suggestion
  "uid": "o11y-cost-estimate",
```

**Comment by Rotfuks** [comment]
@QuantumEnigmaa 

> *   On what time period are the increase and decrease displayed ? Is it the interval selected on default

Yep, it's just the default interval, I didn't set anything specific. But it makes sense compared to the displayed graph

> *   I guess that you're displaying the average number of nodes to ease the calculation of cost based on how much does a node cost (from its machine type) but that might not be straight forward to understand. Could you maybe add some text below to explain why is this metric displayed ?

The average number of nodes is just generally all numbers of nodes in the cluster - that's important because some competitors like DataDog and Dynatrace calculate their price based on the amount of nodes monitored. So it's quite straight forward I think :D That's why the second section is just a collection of other metrics that are useful for calculations, but our own cost calculation just needs the first 4 visualisations from the first section

> *   Could you add a "CPU cores" unit for the `CPU Usage` graph pls ?

It's in the description, that the number is cpu cores - is that enough for you?

**Review by QuantumEnigmaa — APPROVED** [review]

**Pull request merged**